start-page/settings.html

```diff
@@ -1,184 +0,0 @@
-<html>
```

Comment:
Settings is an integral part of the Start Page's functionality, DO NOT delete it, otherwise you'll immediately get a 'Needs changes'.

Reply:
I accidentally removed it.. sorry about that.